packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -990,14 +990,18 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
                     break;
                 }
+                case api.SummaryType.Attachment: {
+                    value = { id: summaryObject.id };
```

Comment:
Curious, can we simply do id = summaryObject.id (and not have value)?
Is this how SPO expects referencing blobs?